src/server/routes.rest

```diff
@@ -0,0 +1,20 @@
+
+
+GET http://localhost:9000/jobs
+
+
+###
+
+GET http://localhost:9000/jobs/1234131231242
+
+###
+
+GET http://localhost:9000/users/63edb27d0e77e161a004824c/jobsApplied
+
+
+###
+
+POST http://localhost:9000/users/63edb27d0e77e161a004824c/jobsApplied
+
+###
+
```

Comment:
What is this file used for?

Reply:
This is used for an extension. The extensions allows us to send requests just like postman. I'll put it in my gitignore file.